sloppy-deployment/entrypoint.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash -eu
+set -x
 
 SLOPPY_OUTPUT_FILE="sloppy.yml"
+safe_yaml="${SLOPPY_SAVE_SLOPPY_YAML:-}"
```

Comment:
We should be consistent and use upper-case variable names.
Furthermore please use the same name for variables as we did before: e.g why not `SLOPPY_SAVE_OUTPUT`?

Reply:
done